lib/action_subscriber/babou.rb

```diff
@@ -0,0 +1,103 @@
+module ActionSubscriber
+  module Babou
+    ##
+    # Class Methods
+    #
+    def self.configure_from_yaml_and_cli(cli_options = {})
+      env = ENV["RAILS_ENV"] || ENV["RACK_ENV"] || ENV["APP_ENV"] || "development"
+
+      yaml_config = {}
+      babou_absolute_config_path = ::File.expand_path(::File.join("config", "babou.yml"))
```

Comment:
I only meant that it would be nice to have the backwards compatibility for
our own internal use so I could move updates up the pipeline even before we
get the new config files in place.

On Mon, Sep 14, 2015 at 5:40 PM, Garrett T <notifications@github.com> wrote:

> In lib/action_subscriber/babou.rb
> <https://github.com/moneydesktop/action_subscriber/pull/18#discussion_r39460579>
> :
>
> > @@ -0,0 +1,103 @@
> > +module ActionSubscriber
> > +  module Babou
> > +    ##
> > +    # Class Methods
> > +    #
> > +    def self.configure_from_yaml_and_cli(cli_options = {})
> > +      env = ENV["RAILS_ENV"] || ENV["RACK_ENV"] || ENV["APP_ENV"] || "development"
> > +
> > +      yaml_config = {}
> > +      babou_absolute_config_path = ::File.expand_path(::File.join("config", "babou.yml"))
>
> @mmmries <https://github.com/mmmries> Why not just rename this to
> action_subscriber.yml right now? I think it would be easy to change the
> config when we switch the CLI away from babou. Doesn't really make sense to
> support code that was never public.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/moneydesktop/action_subscriber/pull/18/files#r39460579>
> .
>